src/controllers/dacpacDialogWebviewController.ts

```diff
@@ -425,6 +440,8 @@ export class DacpacDialogWebviewController extends ReactWebviewPanelController<
             return appResult;
         } catch (error) {
             const errorMessage = error instanceof Error ? error.message : String(error);
+            const errorType = error instanceof Error ? error.constructor.name : typeof error;
+            this.logger.error(`Extract DACPAC operation threw exception: ${errorType}`);
```

Comment:
For the loggers, we can include the full error message `getErrorMessage(err)` because those stay on the user's machine.